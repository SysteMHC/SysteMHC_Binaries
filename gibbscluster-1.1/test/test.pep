TMCIYQSPVRKLKLAC
FKLALFFIIFNKQFHQI
RPPIFIRRLPSCHN
CTVHQLPAIIRPSFW
RARIKTRLF
TDKLYGLSQGY
SDCDQMVIENGILKNK
RGRIYKRSLKLDDLLS
IKRNAVLIDAGLPKLQ
IILAALFMYYTKYW
AVLQSGFRK
DLKVLSHGWAYKA
YVVMMVIFRLMRDT
GYHNSPASFFSSWVQA
SGTNIILAHFLYKEFTMN
KKTVLEFILQKIFKH
VNSPADERAVANFW
QIPAFYTPTARV
MWEPGLIFCHSKKKD
PCYKPESRPFDL
DIKDAVYSICKFY
NHCRPRQIAMSLFIM
EMQHRFPIAKVEPVQKQK
PIDPYVVMQVKVPKIRIF
FRDHGATSIYTIERAPMFW
EKKDFRAYRNALSMPIRMN
VPHISRQRLPWL
LLPTNAVVKMKK
FFVACAPFMSDLQFPMTWH
GDGGTRVAVNKSNKQRHS
RHLTPKFTRMVVSN
WHPAQLFKPLTKKFMD
ANHVGALKISQLQKNFPP
FESMTLVPVLEKALMPK
TRPLRMPAYIRNTLYQA
GTEDAPKRPPSAF
IWYPKGPIRFVLAL
NWCIPKRNRSILP
KLRETGAPLP
CMYTRVYRRRLAQS
HEPVYPGIKVRQLT
FTEAQPKPGTRMVPCT
EWCETLSERISSKRYGRA
PALYSYASAKWW
MENSTMTIREFPRKSYVM
IAPLLSAGIFH
LKVSWRWMVY
ILLFYCPQKKKSQAEFWD
EINEWLSSKDRR
INSNDFPVRPQVPITRCCC
LFRMKAYKIISLKEVEYF
LYDSFTYLCGFIKSLCC
FAGALIRATSTRIGDG
GLSPSKRNQTFIPW
GLPTRCRLEIVHGMG
KFVNSPLQLRFRMII
AENKLYKMRIPRHL
RTLHPFGCKSN
QCRFPASFFIKLFPFM
PERTFGKLPYRNL
YIDQMCVFKFIVAKS
GVFKMGAFMYTKTYD
HEALDGTFQRKYMAF
QRFYPIVIAPAMNEDSH
QRHDTLARSICEK
TALMRWRHPRVVSDS
MYMVYVPHVIEEVMRLSL
KYLYLWSFKPTMQ
RTMSGLPAIVREAIEE
HLVNAISRLRTQKANSIC
SQIQAWPLLPHVIF
EIYALLAVGATKGA
SAIFASSMTKTN
DLFFWYTNEIIGYKVAL
LWRRTKLMSNIKMNC
HQPHRVMANNVKKDIII
KLAQVRRAMCAD
VAMSTWSFYH
RQFVSNNGKVPH
