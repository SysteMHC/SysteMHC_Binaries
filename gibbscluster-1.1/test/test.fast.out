# /usr/cbs/bio/src/gibbscluster-1.0/Downloadables/gibbscluster-1.1/bin/gibbs_cluster_1.1.Linux_x86_64 -fast -s 1 test.pep
# Mon Dec 28 15:34:08 2015
# User: massimo
# PWD : /usr/cbs/bio/src/gibbscluster-1.0/Downloadables/gibbscluster-1.1/test
# Host: Linux padawan 2.6.37.6-0.11-desktop x86_64
# -fast    1                    Run in FAST mode (recommended for large sets)
# -s       1                    Seed [-1] Default, [0] Time [>0] Specific seed
# Command line parameters set to:
#	[-fast]              1                    Run in FAST mode (recommended for large sets)
#	[-l int]             9                    Motif length
#	[-g int]             2                    Number of initial clusters
#	[-trash]             0                    Use trash cluster to remove outliers
#	[-i int]             50                   Number of iterations per train example
#	[-s int]             1                    Seed [-1] Default, [0] Time [>0] Specific seed
#	[-f filename]        0                    File for frequency matrices
#	[-m filename]        0                    File for log-odds matrices
#	[-a filename]        aln.txt              File for multiple alignments
#	[-pc]                0                    Print core of alignment
#	[-c filename]        core.txt             File for core
#	[-alp string]        ARNDCQEGHILKMFPSTWYVBZX Amino acid alphabet
#	[-bg int]            1                    [0] Flat (0.05) [1] Precalculated [2] From data
#	[-base float]        0.000000             Baseline value to discard to trash
#	[-mhc1]              0                    Use peptides of fixed length, no window shift
#	[-ts float]          0.500000             Start temperature
#	[-te float]          0.000010             Final temperature
#	[-nt int]            20                   Number of temperature steps
#	[-ms int]            3                    Max phase shift distance
#	[-fs int]            1000                 Interval between Phase shift moves
#	[-fr int]            10                   Interval between Single sequence moves
#	[-lambda float]      1.000000             Penalty factor for overlapping clusters
#	[-sigma float]       10.000000            Weight on blosum for small cluster confidence
#	[-noph]              0                    Exclude phase shift move
#	[-r float]           5.000000             Scale factor for phase shift moves
#	[-wlc float]         200.000000           Weigth on low count
#	[-blf filename]      /usr/cbs/bio/src/gibbscluster-1.0/Downloadables/gibbscluster-1.1/data/blosum%i.freq_rownorm Real value blosum matrix filename
#	[-blm int]           62                   Blosum matrix number
#	[-cX int]            1                    [0] Count X as A, [1] Exclude X from count
#	[-v]                 0                    Verbose debug mode
#	[-cthr float]        0.620000             Clustering threshold for sequence weighting
#	[-swt int]           0                    [0] 1/ns [1] Clustering [2] No sequence weighting
#	[-w string]          1,3,1,1,1,1,1,1,3    Weigths on positions
#	[-iw]                0                    Include weights on positions
#	[-noc float]         0.000000             Log-odds score assignment to no count elements
#	[-p1a]               0                    Allow only Hydrophobic AA at P1
#	[-p1l filename]      ILVMFYW              Allowed letters at P1 [Only with -p1a option]
#	[-factor float]      2.885400             Scale factor for matrix 2/log2
#	[-cntthr int]        0                    Low bound for amino acids counts to include in matrix
#	[-p1s]               0                    Penalize disallowed P1 amino acids in shift
#	[-pkld]              0                    Print KLD list (file to -kldf)
#	[-kldf string]       kldfq.txt            Print complete list of KLD values to specified file
#	[-kldq int]          1000                 Total datapoints when printing KLDs
# Random seed is 1
# Read 79 elements on seqlist test.pep
# Number of temperature steps 20 dt 0.024999
# Back ground frequences
# 0 A  0.07400
# 1 R  0.05200
# 2 N  0.04500
# 3 D  0.05400
# 4 C  0.02500
# 5 Q  0.03400
# 6 E  0.05400
# 7 G  0.07400
# 8 H  0.02600
# 9 I  0.06800
# 10 L  0.09900
# 11 K  0.05800
# 12 M  0.02500
# 13 F  0.04700
# 14 P  0.03900
# 15 S  0.05700
# 16 T  0.05100
# 17 W  0.01300
# 18 Y  0.03200
# 19 V  0.07300
# Read_realblosum done. Alphabet ARNDCQEGHILKMFPSTWYV
### Initial alignments weighed KLD sum 0.000000

## Sequence alignment for group 0
#Sequence            Alignment	Annotation	Self	BGgroup	BGscore	Corrected
Seq1                 ---NWCIPKRNRSILP---------	Peplist	14.303	-1	0.000	14.303
Seq2                 -------RARIKTRLF---------	Peplist	3.363	-1	0.000	3.363
Seq3                 ---QRFYPIVIAPAMNEDSH-----	Peplist	11.278	1	1.544	9.734
Seq4                 ------KLAQVRRAMCAD-------	Peplist	5.037	1	1.149	3.888
Seq5                 -------GTEDAPKRPPSAF-----	Peplist	3.953	-1	0.000	3.953
Seq6                 --KFVNSPLQLRFRMII--------	Peplist	7.330	1	2.248	5.082
Seq7                 -INSNDFPVRPQVPITRCCC-----	Peplist	9.692	-1	0.000	9.692
Seq8                 FESMTLVPVLEKALMPK--------	Peplist	10.928	1	5.902	5.026
Seq9                 ----VNSPADERAVANFW-------	Peplist	5.769	-1	0.000	5.769
Seq10                ---GLSPSKRNQTFIPW--------	Peplist	7.636	1	1.500	6.137
Seq11                -MYMVYVPHVIEEVMRLSL------	Peplist	8.824	1	0.705	8.118
Seq12                ------SQIQAWPLLPHVIF-----	Peplist	5.609	1	1.869	3.740
Seq13                -CTVHQLPAIIRPSFW---------	Peplist	9.504	1	0.241	9.263
Seq14                -------PERTFGKLPYRNL-----	Peplist	12.820	1	0.394	12.426
Seq15                ---NHCRPRQIAMSLFIM-------	Peplist	9.098	-1	0.000	9.098
Seq16                -------PCYKPESRPFDL------	Peplist	6.412	-1	0.000	6.412
Seq17                ----IWYPKGPIRFVLAL-------	Peplist	5.033	-1	0.000	5.033
Seq18                -RTMSGLPAIVREAIEE--------	Peplist	8.768	1	2.766	6.002
Seq19                -EKKDFRAYRNALSMPIRMN-----	Peplist	6.465	1	4.068	2.397
Seq20                ----MENSTMTIREFPRKSYVM---	Peplist	5.320	1	2.841	2.479
Seq21                -----GLPTRCRLEIVHGMG-----	Peplist	8.103	-1	0.000	8.103
Seq22                -----RQFVSNNGKVPH--------	Peplist	3.523	1	1.141	2.382
Seq23                ------FFVACAPFMSDLQFPMTWH	Peplist	4.210	1	1.570	2.641
Seq24                ---RHLTPKFTRMVVSN--------	Peplist	8.701	1	1.839	6.862
Seq25                --FTEAQPKPGTRMVPCT-------	Peplist	7.270	-1	0.000	7.270
Seq26                FRDHGATSIYTIERAPMFW------	Peplist	7.792	1	3.067	4.725
Seq27                ------VPHISRQRLPWL-------	Peplist	12.831	1	0.443	12.388
Seq28                ---QCRFPASFFIKLFPFM------	Peplist	7.781	1	2.563	5.219
Seq29                -----LLPTNAVVKMKK--------	Peplist	6.603	1	1.161	5.442
Seq30                -----HEPVYPGIKVRQLT------	Peplist	9.040	1	5.315	3.726
Seq31                -------PIDPYVVMQVKVPKIRIF	Peplist	7.576	1	6.347	1.229
Seq32                ----YVVMMVIFRLMRDT-------	Peplist	2.611	1	1.973	0.638
Seq33                -EMQHRFPIAKVEPVQKQK------	Peplist	6.651	1	4.970	1.681
Seq34                -TRPLRMPAYIRNTLYQA-------	Peplist	9.159	1	5.612	3.547
Seq35                ------RPPIFIRRLPSCHN-----	Peplist	11.121	1	5.444	5.677
Seq36                -----QIPAFYTPTARV--------	Peplist	6.001	1	4.653	1.347
Seq37                ------KLRETGAPLP---------	Peplist	4.287	1	3.497	0.790
Seq38                -----WHPAQLFKPLTKKFMD----	Peplist	9.566	1	8.053	1.514
                     .......*********
## Sequence alignment for group 1
#Sequence            Alignment	Annotation	Self	BGgroup	BGscore	Corrected
Seq1                 -------DLFFWYTNEIIGYKVAL	Peplist	5.296	-1	0.000	5.296
Seq2                 ---QRHDTLARSICEK--------	Peplist	7.071	-1	0.000	7.071
Seq3                 --YIDQMCVFKFIVAKS-------	Peplist	9.339	-1	0.000	9.339
Seq4                 ------TALMRWRHPRVVSDS---	Peplist	8.897	0	2.271	6.626
Seq5                 ---HLVNAISRLRTQKANSIC---	Peplist	8.478	0	3.173	5.305
Seq6                 --SDCDQMVIENGILKNK------	Peplist	4.789	0	0.504	4.285
Seq7                 -------AVLQSGFRK--------	Peplist	9.508	-1	0.000	9.508
Seq8                 ----FKLALFFIIFNKQFHQI---	Peplist	8.684	-1	0.000	8.684
Seq9                 ----DLKVLSHGWAYKA-------	Peplist	7.647	-1	0.000	7.647
Seq10                --ANHVGALKISQLQKNFPP----	Peplist	4.264	0	2.979	1.286
Seq11                ----EIYALLAVGATKGA------	Peplist	7.927	-1	0.000	7.927
Seq12                --LWRRTKLMSNIKMNC-------	Peplist	3.669	0	1.607	2.061
Seq13                ---EWCETLSERISSKRYGRA---	Peplist	8.241	0	1.940	6.301
Seq14                ------ILLFYCPQKKKSQAEFWD	Peplist	6.011	0	2.742	3.269
Seq15                --MWEPGLIFCHSKKKD-------	Peplist	8.369	-1	0.000	8.369
Seq16                -----KKTVLEFILQKIFKH----	Peplist	9.629	0	1.001	8.628
Seq17                -----TDKLYGLSQGY--------	Peplist	4.361	-1	0.000	4.361
Seq18                ---DIKDAVYSICKFY--------	Peplist	6.741	0	1.177	5.563
Seq19                -------GVFKMGAFMYTKTYD--	Peplist	4.042	0	0.077	3.965
Seq20                --IKRNAVLIDAGLPKLQ------	Peplist	5.876	0	3.061	2.814
Seq21                ------PALYSYASAKWW------	Peplist	10.358	0	0.685	9.673
Seq22                ---HQPHRVMANNVKKDIII----	Peplist	7.658	0	2.028	5.631
Seq23                ------SAIFASSMTKTN------	Peplist	10.490	0	5.120	5.369
Seq24                ---SGTNIILAHFLYKEFTMN---	Peplist	5.626	0	2.475	3.151
Seq25                ---IILAALFMYYTKYW-------	Peplist	9.135	0	1.404	7.731
Seq26                GDGGTRVAVNKSNKQRHS------	Peplist	6.689	-1	0.000	6.689
Seq27                -----TMCIYQSPVRKLKLAC---	Peplist	8.901	0	5.899	3.001
Seq28                -------EINEWLSSKDRR-----	Peplist	5.202	-1	0.000	5.202
Seq29                ----FAGALIRATSTRIGDG----	Peplist	5.572	0	1.364	4.208
Seq30                -------VAMSTWSFYH-------	Peplist	8.531	-1	0.000	8.531
Seq31                -------KYLYLWSFKPTMQ----	Peplist	10.563	0	3.543	7.020
Seq32                ------RTLHPFGCKSN-------	Peplist	3.119	-1	0.000	3.119
Seq33                -----RGRIYKRSLKLDDLLS---	Peplist	5.196	0	3.314	1.882
Seq34                --LYDSFTYLCGFIKSLCC-----	Peplist	2.575	0	0.788	1.787
Seq35                ---LFRMKAYKIISLKEVEYF---	Peplist	6.370	0	3.214	3.156
Seq36                ---CMYTRVYRRRLAQS-------	Peplist	5.565	0	4.539	1.026
Seq37                GYHNSPASFFSSWVQA--------	Peplist	7.827	0	4.416	3.411
Seq38                ------LKVSWRWMVY--------	Peplist	4.599	0	2.669	1.930
Seq39                -----IAPLLSAGIFH--------	Peplist	4.234	0	2.571	1.663
Seq40                -----HEALDGTFQRKYMAF----	Peplist	5.019	0	2.664	2.355
Seq41                ----AENKLYKMRIPRHL------	Peplist	9.333	0	5.574	3.759
                     .......*********
# group 0, size 38: Final KLD sum after 79000 epochs - 5.449882
# group 1, size 41: Final KLD sum after 79000 epochs - 5.087808
#
# Final number of non-empty clusters: 2

### Final KLD sum after 79000 iterations: 5.261970
